include/swift/AST/Identifier.h

```diff
@@ -206,15 +208,45 @@ namespace swift {
 
 /// Wrapper that may either be an Identifier or a special name
 /// (e.g. for subscripts)
+///
+/// The raw values of this enum are used in serialization by both
```

Comment:
This comment should no longer be necessary, right? (Also, it belongs on the enum, not the class!)